src/template.ts

```diff
@@ -165,6 +165,18 @@ template.innerHTML = `
       transform: translateY(-100%);
     }
 
+    @keyframes wiggle {
+      0%, 50%, 100% {
+        transform: translateX(0%);
+      }
+      25% {
+        transform: translateX(-5%);
+      }
+      75% {
+        transform: translateX(5%);
```

Comment:
Could we change this animation to be intermittent? I think it would be more pleasing to the eye if it were a bit shorter with a few seconds of space in between that didn't show the SVG.

Reply:
Agreed with the intermittent animations, after a bit of searching it doesn't seem there's a way to pause animation between iterations through CSS.

There's a few alternatives:
* [placing a pause within the animation keyframes](https://css-tricks.com/css-keyframe-animation-delay-iterations/)
* [performing a few iterations of the animation each with a delay on them](http://www.standardista.com/animation-iteration-delay/)

In terms of the UX, what do you think about this animation?

![model](https://user-images.githubusercontent.com/42545625/65969113-38907600-e432-11e9-8f8b-62f39c73eb4b.gif)

Reply:
Here's a preview of what we were thinking about the fine tuned adjustments for the animation: https://translucent-lunch.glitch.me

Reply:
@elalish We're feeling pretty good about the latest animation that @maaslalani just posted.

The symmetry is really important to us as it shows a good range of motion, and makes for a good flow.

In a subsequent PR we also have some changes to the hand svg based on some usability research we did.

Reply:
That animation looks really good to me! Thanks for the effort.

Reply:
@pushmatrix if we changed the animation in a subsequent change (perhaps the timing function, or the duration), would you consider that a breaking change?